src/main/java/net/pms/newgui/NavigationShareTab.java

```diff
@@ -383,7 +384,7 @@ public void itemStateChanged(ItemEvent e) {
 
 		// Enable the Media Library
 		cacheenable = new JCheckBox(Messages.getString("NetworkTab.EnableMediaLibrary"), configuration.getUseCache());
-		cacheenable.setToolTipText(Messages.getString("FoldTab.ShowMediaLibraryFolderTooltip"));
+		cacheenable.setToolTipText(toolTipMessage("FoldTab.ShowMediaLibraryFolderTooltip", "FoldTab.ShowMediaLibraryFolderTooltipNote"));
```

Comment:
@ahmedadan why do you not use the `Enabled` here as I introduced? Using this can simplify the message.properties to not translate this general setting for all relevant notes again.

Reply:
Sorry for double-reviewing, I hadn't seen yours :)

Reply:
I see, I can simplify this as well.

Reply:
@SubJunk I was accidentally faster than you 😃